packages/pricing/src/services/money-amount.ts

```diff
@@ -72,10 +72,6 @@ export default class MoneyAmountService<
       config
     )
 
-    if (filters.id) {
```

Comment:
todo: I think we should remove this method as it does not do anything that calling buildQuery and returning the result